Polyhedron/demo/Polyhedron/Scene_triangulation_3_item.cpp

```diff
@@ -353,6 +353,7 @@ struct Scene_triangulation_3_item_priv {
     is_aabb_tree_built = false;
     alphaSlider = NULL;
     is_filterable = true;
+    memset(visible_biteset, 0xFFFF, Scene_triangulation_3_item::number_of_bitset * sizeof(uint)); // all bits set to 1
```

Comment:
There is a warning on both uses of `memset`.
```
/home/lrineau/Git/cgal-master/Polyhedron/demo/Polyhedron/Scene_triangulation_3_item.cpp: In member function ‘void Scene_triangulation_3_item_priv::init_default_values()’:
/home/lrineau/Git/cgal-master/Polyhedron/demo/Polyhedron/Scene_triangulation_3_item.cpp:373:11: warning: ‘void* memset(void*, int, size_t)’ writing to an object of non-trivial type ‘class std::bitset<32>’; use assignment instead [-Wclass-memaccess]
  373 |     memset(visible_biteset, 0xFFFF, Scene_triangulation_3_item::number_of_bitset * sizeof(uint)); // all bits set to 1
      |     ~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/boost/detail/container_fwd.hpp:96,
                 from /usr/include/boost/container_hash/extensions.hpp:22,
                 from /usr/include/boost/container_hash/hash.hpp:766,
                 from /usr/include/boost/functional/hash.hpp:6,
                 from /home/lrineau/Git/cgal-master/Stream_support/include/CGAL/IO/Color.h:23,
                 from /home/lrineau/Git/cgal-master/Stream_support/include/CGAL/IO/io.h:24,
                 from /home/lrineau/Git/cgal-master/Kernel_23/include/CGAL/Bbox_3.h:21,
                 from /home/lrineau/Git/cgal-master/Three/include/CGAL/Three/Scene_interface.h:46,
                 from /home/lrineau/Git/cgal-master/Polyhedron/demo/Polyhedron/Scene.h:6,
                 from /home/lrineau/Git/cgal-master/Polyhedron/demo/Polyhedron/Scene_triangulation_3_item.cpp:1:
/usr/include/c++/13/bitset:797:11: note: ‘class std::bitset<32>’ declared here
  797 |     class bitset
      |           ^~~~~~
/home/lrineau/Git/cgal-master/Polyhedron/demo/Polyhedron/Scene_triangulation_3_item.cpp: In member function ‘void Scene_triangulation_3_item::resetVisibleSubdomain()’:
/home/lrineau/Git/cgal-master/Polyhedron/demo/Polyhedron/Scene_triangulation_3_item.cpp:2157:9: warning: ‘void* memset(void*, int, size_t)’ writing to an object of non-trivial type ‘class std::bitset<32>’; use assignment instead [-Wclass-memaccess]
 2157 |   memset(d->visible_biteset, 0xFFFF, number_of_bitset * sizeof(uint));
      |   ~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/13/bitset:797:11: note: ‘class std::bitset<32>’ declared here
  797 |     class bitset
      |           ^~~~~~
```
It is not allowed `memset` in a `std::bitset` because the implementation maybe use a different representation.

Use `std::fill` instead.


.. Actually:
  - `visible_biteset` should be a `std::array`,
  - then use `visible_biteset.fill(0xFFFF)`
  - and that variable should by renamed to `visible_bitset` (`bit` not `bite`).